package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@interlay/polkabtc",
-  "version": "0.15.6",
+  "version": "0.16.0",
```

Comment:
0.15.7 will soon be merged, hope it won't cause a conflict

Reply:
yeah, no worries I'll fix that